api/Element.json

```diff
@@ -6203,10 +6204,10 @@
             "description": "Returns a <code>Promise</code>",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": true
```

Comment:
This should say 69. https://storage.googleapis.com/chromium-find-releases-static/f24.html#f24f5da91960f0afa3b0e325ebab7c81ece796d4

Reply:
As should Chrome Android and Android WebView.